.gitignore

```diff
@@ -284,5 +284,9 @@ tests/tests_unit/pytest-project/
 my_project/
 *.bck
 
+# Hide environments used for migration testing
+tests_migrations/.venv*
```

Comment:
I need the same for .venv.* across many folders (creating venv in tests), maybe add `**/.venv*`